docs/source/tutorials.rst

```diff
@@ -16,26 +18,51 @@ The notebook also shows examples of performing:
 
 Multi-band images are plotted using functions defined in *btk.plot_utils*
 
-Run btk with user-input custom functions (*custom_sampling_function.ipynb*).
-----------------------------------------------------------------------------
-btk generates postage stamp images of blend scenes from an input catalog. How a particular scene is sampled from this catalog is defined in a function input to *btk.create_blend_generator.generate*.
-This `notebook <https://github.com/LSSTDESC/BlendingToolKit/blob/master/notebooks/custom_sampling_function.ipynb>`_ demonstrates how users can define their own sampling function to draw blend image scenes.
 
-Also shown is an example of how a user-defined custom function can be used to generate different observing conditions. This enables on-the-fly generation of blend scene images with different noise levels and observing PSFs.
+Tests
+------
+
+Importing relevant modules
+
+.. jupyter-execute::
```

Comment:
Interesting, are you needing to type the Jupyter notebook contents directly into this `.rst` file?

Reply:
Basically yes, the jupyter-sphinx library import those, create an actual notebook, executes it and put the results back in. It is not interactive though but I believe it is enough for the documentation.

Reply:
(I started rewriting an actual notebook and will transfer the content in the doc afterwards)

Reply:
sounds good, are you also able to use an existing notebook instead of having to rewrite it inside the `.rst` document?

Reply:
Well I have to rewrite it but it is pretty straightforward to do (mostly copy and paste). I think there is a way of just integrating a notebook inside the doc directly but I think it is better as it is now.